lib/user.rb

```diff
@@ -0,0 +1,6 @@
+class User
+  attr_accessor :first_name, :last_name
+
+  def initialize
+  end
+end
```

Comment:
You don't need an initializer if you're not doing anything during initialization. 
```suggestion
class User
  attr_accessor :first_name, :last_name
end
```